.github/workflows/helm-chart.yml

```
@@ -36,7 +36,7 @@ jobs:
 
       - name: Set helm chart appVersion
         run: |
-          sed -ie 's#Version.props#"${VERSION_PROPS}"#g' charts/garnet/Chart.yaml
+          sed -i -e 's#Version.props#"'${VERSION_PROPS}"'#g' charts/garnet/Chart.yaml
```

There is a error is this, check this fix:
```suggestion
          sed -i -e 's#Version.props#"'${VERSION_PROPS}'"#g' charts/garnet/Chart.yaml
```